API/Controllers/TournamentsController.cs

```diff
@@ -60,7 +52,7 @@ public async Task<IActionResult> CreateAsync([FromBody] TournamentSubmissionDTO
         if (await tournamentsService.ExistsAsync(tournamentSubmission.Name, tournamentSubmission.Ruleset))
         {
             return BadRequest(
-                $"A tournament with name {tournamentSubmission.Name} for ruleset {tournamentSubmission.Ruleset} already exists"
+                $"A tournament with name: {tournamentSubmission.Name} for ruleset: {tournamentSubmission.Ruleset} already exists"
```

Comment:
```suggestion
                $"A tournament with name '{tournamentSubmission.Name}' for ruleset '{tournamentSubmission.Ruleset}' already exists"
```

This seems like it would read nicer in a log. Colons don't work here.